### ISSUE TITLE
Message queueing should be one of the tips, esp if a user hasn't done it yet

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#293536]
Otherwise it's not discoverable, esp for screen reader users https://github.com/microsoft/vscode/issues/293535
